fe/fe-core/src/test/java/com/starrocks/catalog/FunctionSetTest.java

```diff
@@ -242,4 +243,29 @@ public void testPolymorphicFunction() {
         Assert.assertEquals(Type.INT, fn.getReturnType());
         Assert.assertEquals(new ArrayType(Type.NULL), fn.getArgs()[0]);
     }
+
+    @Test
+    public void testPolymorphicTVF() {
+        // First two columns of the result are polymorphic (derived from argument type), but the last argument is of a
+        // "concrete" type BIGINT, which is retained in the resolved function.
+        TableFunction polymorphicTVF =
+                new TableFunction(new FunctionName("three_column_tvf"), Lists.newArrayList("a", "b", "c"),
+                        Lists.newArrayList(Type.ANY_ARRAY),
+                        Lists.newArrayList(Type.ANY_ELEMENT, Type.ANY_ELEMENT, Type.BIGINT));
+
+        functionSet.addBuiltin(polymorphicTVF);
+
+
```

Comment:
Remove this empty line.

Does two column tvf also work?

Reply:
added a test